packages/nimble/R/MCMC_samplers.R

```diff
@@ -437,6 +437,11 @@ sampler_slice <- nimbleFunction(
         adaptInterval <- if(!is.null(control$adaptInterval)) control$adaptInterval else 200
         width         <- if(!is.null(control$sliceWidth))    control$sliceWidth    else 1
         maxSteps      <- if(!is.null(control$sliceMaxSteps)) control$sliceMaxSteps else 100
+        maxContractions        <- if(!is.null(control$maxContractions))
+                                      control$maxContractions else 100
```

Comment:
@paciorek Should we choose a larger default?  I'm thinking we want to be conservative, with the goal to avoid a infinite loop in invalid or pathological cases but also to avoid false positives.